Jenkinsfile

```diff
@@ -44,11 +44,6 @@ podTemplate(
               sh 'npm run lint'
             }
           },
```

Comment:
`removal of 'unit test'`
does this need to be re-instated or are we going to wait for the browserstack replacement?

Reply:
Fixed on rebase